uber/templates/budget/index.html

```diff
@@ -1,4 +1,5 @@
-{% extends "base-admin.html" %}
+{% set admin_area=True %}
```

Comment:
If we're setting admin_area in individual pages, is the base-admin still needed? Was the goal to refactor base-admin out entirely, and that's just not quite completed yet?